### ISSUE TITLE
REPRODUCE command has double quotes around jvm options

### ISSUE THREAD

**Issue by brwe** [elastic/elasticsearch#15048]
See for example http://build-us-00.elastic.co/job/es_core_2x_window-2008/265/consoleFull
```
mvn verify -Pdev -Dskip.unit.tests -pl org.elasticsearch.qa.backwards:2.0.0 -Dtests.seed=9772A1E19F8B6B15 -Dtests.class=org.elasticsearch.search.aggregations.bucket.SignificantTermsBackwardCompatibilityIT -Dtests.method="testBucketStreaming" -Des.logger.level=DEBUG -Dtests.nightly=false -Dtests.client.ratio=0.0 -Dtests.heap.size=512m -Des.node.mode=network -Dtests.jvm.argline=""-server -XX:+UseSerialGC -XX:+UseCompressedOops -Djava.net.preferIPv4Stack=true"" -Dtests.locale=fr -Dtests.timezone=America/Cuiaba

```

but should be 

```
... -Dtests.jvm.argline="-server -XX:+UseSerialGC -XX:+UseCompressedOops -Djava.net.preferIPv4Stack=true" ...

**Comment by jobin-jose** [comment]
This fixes the additional double quotes in jvm argline which appears in REPRODUCE WITH

**Comment by jobin-jose** [comment]
https://github.com/jobin-jose/elasticsearch/commit/6863d93ac38f28919ac3fd047583ba59b5ccd071

**Comment by clintongormley** [comment]
Closed by #16407
